### PR TITLE
CSS changes for better distinction between user name and full name in user list

### PR DESCRIPTION
**Pull request by Jerome-Herbinet**
Signed-off-by: Jérôme Herbinet jerome.herbinet@arawa.fr

Signed-off-by: Jérôme Herbinet <33763786+Jerome-Herbinet@users.noreply.github.com>

### CODE REVIEW

**Review by szaimen — APPROVED** [review]
Lgtm but didn't test

**Review by skjnldsv — APPROVED** [review]

File: apps/settings/css/settings.scss [hunk 1]
```diff
@@ -1634,7 +1634,7 @@ doesnotexist:-o-prefocus, .strengthify-wrapper {
 						flex-grow: 1;
 					}
 				}
-
+
```
[the remark this answers is not in the document]
Comment (Jerome-Herbinet) [reply apps/settings/css/settings.scss:1637]: I'll do it.

File: apps/settings/css/settings.scss [hunk 2]
```diff
@@ -1649,6 +1649,14 @@ doesnotexist:-o-prefocus, .strengthify-wrapper {
 					-webkit-line-clamp: 2;
 					-webkit-box-orient: vertical;
 				}
+				
+				&.name {
+					font-style: italic;
+					color: var(--lt-color-text-very-light);
+					.displayName.subtitle {
+						font-style: normal;
+					}
```
Comment (szaimen) [inline apps/settings/css/settings.scss:1658]: Here seems to be some whitespace behind the bracket that should be removed

File: apps/settings/css/settings.scss [hunk 3]
```diff
@@ -1649,6 +1649,14 @@ doesnotexist:-o-prefocus, .strengthify-wrapper {
...
 					-webkit-box-orient: vertical;
 				}
+				
+				&.name {
+					font-style: italic;
+					color: var(--lt-color-text-very-light);
+					.displayName.subtitle {
+						font-style: normal;
+					}					
+				}
```
Comment (szaimen) [inline apps/settings/css/settings.scss:1659]: Here as well?

File: apps/settings/css/settings.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jerome-Herbinet) [reply apps/settings/css/settings.scss]: @szaimen I solved it, but I did have to add "Signed-off-by: ..." in my commit's description ?

[the remark this answers is not in the document]
Comment (skjnldsv) [reply apps/settings/css/settings.scss]: You have to sign your commits yes @Jerome-Herbinet 
Sign off your commits using `git commit -sm "Your commit message"`

File: apps/settings/css/settings.scss (hunk 2 above)
Comment (Jerome-Herbinet) [reply apps/settings/css/settings.scss]: Done

File: apps/settings/css/settings.scss (hunk 3 above)
Comment (Jerome-Herbinet) [reply apps/settings/css/settings.scss]: Done

File: apps/settings/css/settings.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jerome-Herbinet) [reply apps/settings/css/settings.scss]: Thanks @skjnldsv ;  i've reverted my commit, and i have done a new one ... which is signed.
@szaimen it's OK

**Comment by Jerome-Herbinet** [comment]
What is in the commit branch ? is this normal ? any problem ?
[3rdparty](https://github.com/nextcloud/server/pull/33669/files#diff-c16cb0249ed0e5de41d9e31fd3ca629585140a3a21b90b5ddffb467e8463d2a1)
Viewed
Submodule [3rdparty](https://github.com/nextcloud/3rdparty) updated 164 files

**Review by jancborchardt — CHANGES_REQUESTED** [review]
Good idea! Some design feedback:
- [ ] We never use italic cause it is not so easily legible
- [ ] Instead, we could bold the display name
- [ ] Display name should show in first row, then account name, so it doesn't look weird with a bold subline
- [ ] That variable looks non-standard, we only use either color-main-text or color-text-maxcontrast. But if we bold the display name, we don't need to reduce the color of the account name.

**Comment by szaimen** [comment]
> What is in the commit branch ? is this normal ? any problem ? [3rdparty](https://github.com/nextcloud/server/pull/33669/files#diff-c16cb0249ed0e5de41d9e31fd3ca629585140a3a21b90b5ddffb467e8463d2a1) Viewed Submodule [3rdparty](https://github.com/nextcloud/3rdparty) updated 164 files

This change needs to be reverted. See https://stackoverflow.com/questions/30275691/removing-subproject-commit-from-github/30275888#30275888

**Comment by Jerome-Herbinet** [comment]
> 

@szaimen I think I've removed it ; can you tell me if it's OK ?

**Comment by Jerome-Herbinet** [comment]
> Good idea! Some design feedback:
> 
>     * [ ]  We never use italic cause it is not so easily legible
> 
>     * [ ]  Instead, we could bold the display name
> 
>     * [ ]  Display name should show in first row, then account name, so it doesn't look weird with a bold subline
> 
>     * [ ]  That variable looks non-standard, we only use either color-main-text or color-text-maxcontrast. But if we bold the display name, we don't need to reduce the color of the account name.

@jancborchardt thanks for your suggestions ; I'll take care of it.

**Comment by Jerome-Herbinet** [comment]
* following

File: .gitmodules [hunk 4]
```diff
@@ -1,3 +0,0 @@
-[submodule "3rdparty"]
-	path = 3rdparty
-	url = https://github.com/nextcloud/3rdparty.git
```
Comment (szaimen) [inline .gitmodules:3]: hm... seems like following my link removed too much :/

@skjnldsv do you know how to fix that?

**Comment by szaimen** [comment]
@Jerome-Herbinet thanks! Can you please post a new screenshot? That would be great!

**Comment by Jerome-Herbinet** [comment]
> @Jerome-Herbinet thanks! Can you please post a new screenshot? That would be great!

@szaimen here it is :

**Comment by szaimen** [comment]
cc @jancborchardt

**Comment by jancborchardt** [comment]
@Jerome-Herbinet almost :) could you:
- Unbold the account name, as the display name is already bold
- Put display name in the first line, accoint in the second (not soo important if difficult)

File: .gitmodules (hunk 4 above)
Comment (skjnldsv) [reply .gitmodules:3]: Nope :(

Comment (Jerome-Herbinet) [reply .gitmodules:3]: @StCyr une idée ?

**Comment by Jerome-Herbinet** [comment]
> @Jerome-Herbinet almost :) could you:
> 
>     * Unbold the account name, as the display name is already bold
> 
>     * Put display name in the first line, accoint in the second (not soo important if difficult)

@jancborchardt in fact the text isn't bold ; the other screenshot boldness is certainly due to the default font of my Ubuntu Laptop.

Here is a screenshot of the same NC instance, under Manjaro. 

scr

So do you want me to bold something ?

And about putting the display name in first position, I won't be able to do it my self (sorry).

**Comment by jancborchardt** [comment]
@Jerome-Herbinet ah got it, thanks for the screenshot! Then I'd only say it would be nice to bold the display name (best use a `<strong>` tag for semantic markup).

**Comment by Jerome-Herbinet** [comment]
> @Jerome-Herbinet ah got it, thanks for the screenshot! Then I'd only say it would be nice to bold the display name (best use a `\<strong\>` tag for semantic markup).

Done @jancborchardt 

Display name is bold ... event if on my "day" computer, difference between bold and not bold texte isn't very visible ... (with "strong") and display name and user name are inverted now.

**Comment by Jerome-Herbinet** [comment]
As a newbie as code contributor, I made an error, by committing [5f9962d](https://github.com/Jerome-Herbinet/server/commit/5f9962d427863f635a4d1d2ac93e1a884e3d8808) ... I think I've cancelled it, can you confirm ? (sorry for the mess, I'm learning).

**Comment by Jerome-Herbinet** [comment]
I'm concerned about submodule problem ; do you have a tip to solve it ?

**Comment by Jerome-Herbinet** [comment]
Hello,
If the Submodule problem is not solvable, do you advise me to close this PR and to create a new one (hopping that it will work better) ?

**Comment by szaimen** [comment]
> If the Submodule problem is not solvable, do you advise me to close this PR and to create a new one (hopping that it will work better) ?

Yesy that is probably a good idea. Thanks!

**Comment by Jerome-Herbinet** [comment]
> > If the Submodule problem is not solvable, do you advise me to close this PR and to create a new one (hopping that it will work better) ?
> 
> Yesy that is probably a good idea. Thanks!

OK @szaimen, I close it and tell you when the new PR is created (I'll cross the fingers).

**Pull request closed without merging**